docs/troubleshooting.mdx

```diff
@@ -9,6 +9,8 @@ import TabItem from '@theme/TabItem';
 # Troubleshooting
 ## Panel errors
 
+If you see a `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
```

Comment:
Just like i did first in https://github.com/pelican-dev/docs/pull/34 but you told me to move 500 lower

Reply:
No, I literally wrote `Please move the 502 error and CSRF error to the "common errors" section. The introductory text should stay.*`